src/client/pages/Section/Contacts/Contacts.tsx

```diff
@@ -22,13 +22,15 @@ const Contacts: React.FC<Props> = (props: Props) => {
   const gridTemplateColumns = useMemo(() => `12ch repeat(${columns.length - 1}, 1fr)`, [columns.length])
 
   return (
-    <TableNodeExt
-      columns={columns}
-      data={contacts}
-      disabled={disabled}
-      gridTemplateColumns={gridTemplateColumns}
-      onChange={onChange}
-    />
+    <div className="fra-table__container">
```

Comment:
To get correct margins and padding for Contacts container, otherwise these would almost overlap:
![image](https://github.com/openforis/fra-platform/assets/5508251/74c51496-af2b-459e-bc82-2252ada44176)